docs/articles/infrastructure/ci-cd/freeipa.rst

```diff
@@ -316,4 +490,76 @@ Try to identify an LDAP user:
     # Should return "no such user"
     id someldapuser
 
+Replica
+^^^^^^^
+
+If any of the steps fails during installation you need to run 
+``ipa-server-install --uninstall`` a few times to clean the system. 2-3 should
+do it. After that you need to re-enroll the client before continuing with
+``ipa-client-install --enable-dns-updates --force-join`` (Don't forget to
+re-enable mkhomedir).
+
+If you find yourself re-installing FreeIPA server you will run into certificate
+problems when accessing the Web UI. This is because the certificate server
+doesn't randomize the ID:s of the generated certificates. So the browser won't
+accept seeing a different certificate with the same Domain, Subject and Serial
+Number. To mitigate this you can either pass the an argument during installation
+to change the Subject or configure the Dogtag server to randomize ID:s.
+
+Set the subject:
+
+.. code-block:: bash
+
+    ipa-server-install --setup-dns --hostname=ipa.example.com.com --realm=IPA.EXAMPLE.COM --domain=ipa.example.com --subject="O=EXAMPLE.COM 20181112"
```

Comment:
"hostname=ipa.example.com.com" is this extra "com" on purpose?

Reply:
Good catch